devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -1211,31 +1237,21 @@ void HumanStateProvider::run()
 
 void HumanStateProvider::impl::ereaseSecondaryCalibration(const std::string& linkName)
```

Comment:
Spell correction : erase

Reply:
Done in https://github.com/robotology/human-dynamics-estimation/pull/211/commits/e7bbccb8cef6e44580587fa63930c23e05b4bb51